api/metrics/v1/metrics.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package clutch.metrics.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/metrics/v1;metricsv1";
+
+import "validate/validate.proto";
+import "api/v1/annotations.proto";
+import "google/api/annotations.proto";
+
+service MetricsAPI {
+  rpc GetMetrics(GetMetricsRequest) returns (GetMetricsResponse) {
+    option (google.api.http) = {
+      post : "/v1/metrics/getMetrics"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = READ;
+  }
+}
+
+message Query {
+  // Prometheus expression query string
+  string query = 1 [ (validate.rules).string.min_len = 1 ];
+  // inclusive
+  int64 start_time_ms = 2;
+  // inclusive
+  int64 end_time_ms = 3;
+  // time step that starts from, a nice default is 1 minute (60000)
+  int64 step_ms = 4;
```

Comment:
Same question mike had above, should this just default to 1 minute?

Reply:
I'm not sure what you mean here. Proto3 does not allow default values, unless I'm missing something?

Reply:
yeah I think the only workaround is making it an optional value but even then I think this reads easier